### PR TITLE
[SPARK-39596][INFRA][FOLLOWUP] Install `mvtnorm` and `statmod` at linter job

### PR DESCRIPTION
**Pull request by dongjoon-hyun**

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
cc @LuciferYang

**Comment by dongjoon-hyun** [comment]
I'll convert this to normal after passing CI completely.

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -507,7 +506,7 @@ jobs:
         python3.9 -m pip install 'docutils<0.18.0' # See SPARK-39421
         apt-get update -y
         apt-get install -y ruby ruby-dev
-        Rscript -e "install.packages(c('devtools', 'testthat', 'knitr', 'rmarkdown', 'markdown', 'e1071', 'roxygen2'), repos='https://cloud.r-project.org/')"
+        Rscript -e "install.packages(c('devtools', 'testthat', 'knitr', 'rmarkdown', 'markdown', 'e1071', 'roxygen2', 'ggplot2', 'mvtnorm', 'statmod'), repos='https://cloud.r-project.org/')"
```
Comment (LuciferYang) [inline .github/workflows/build_and_test.yml:509]: should pass ~

https://github.com/LuciferYang/spark/runs/7052537883?check_suite_focus=true 

My test already has been successful

**Review by LuciferYang — APPROVED** [review]
LGTM +1

**Review by zhengruifeng — APPROVED** [review]
LGTM if tests pass

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Comment by HyukjinKwon** [comment]
Merged to branch-3.3 too.

**Comment by dongjoon-hyun** [comment]
Oh, thank you, @LuciferYang , @zhengruifeng , @HyukjinKwon .

**Pull request closed without merging**